### PR TITLE
DOCS: Add Clear Container with Docker install doc

### PR DESCRIPTION
**Pull request by grahamwhaley**
Add installation notes for installing Docker and Clear Containers into
a Clear Linux distribution.

Note: the ReST style 'attention' boxes do not show as boxes on
githubs markdown web rendering, but show fine when using rst2html

### CODE REVIEW

File: INSTALL_DOCKER.rst [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+Download, build and install the clr-oci-runtime from source:
+
+  ::
+
+    cor_source=${HOME}/clr-oci-runtime
+    git clone https://github.com/01org/clr-oci-runtime.git $cor_source
+    cd $cor_source
+    autoreconf -fvi
+    bash autogen.sh --disable-cppcheck --disable-valgrind
```
Comment (grahamwhaley) [inline INSTALL_DOCKER.rst]: I think we leave it as is - and then we'll have to work out the clash for the users who have installed from source.

File: INSTALL_DOCKER.rst [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+      which clr-oci-runtime
+      #typically /usr/bin/clr-oci-runtime
+
+Then edit the Docker_ systemd unit file ExecStart to make `Clear Containers`_ the default runtime.
+
+  ::
+
+    Edit: /usr/lib/systemd/system/docker-upstream.service
+    ExecStart=/usr/bin/dockerd-upstream --add-runtime cor=/usr/bin/clr-oci-runtime--default-runtime=cor -H fd://
```
Comment (grahamwhaley) [inline INSTALL_DOCKER.rst]: 'simply'? :-)  I think it is just personal preference. For me I actually prefer the 'Edit:' version, as it is clearer to me that we are doing an edit and what it is adding.

Comment (jodh-intel) [inline INSTALL_DOCKER.rst]: Fair point :)

I've now fixed this in master, but note the missing space on line 202 before "--default-runtime=cor".

Comment (grahamwhaley) [inline INSTALL_DOCKER.rst]: Just so we get this clear - the action is to *add* arguments to the existing ExecStart line, yes? In which case the text needs to be a little clearer. Are you OK clarifying that @jodh-intel ?

**Pull request merged**